pkgs/applications/editors/fte/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, unzip, perl, libX11, libXpm, gpm, ncurses, slang }:
+
+stdenv.mkDerivation rec {
+  name = "fte-0.50.02";
+
+  buildInputs = [ unzip perl libX11 libXpm gpm ncurses slang ];
+
+  ftesrc    = fetchurl {
+                url = "mirror://sourceforge/fte/fte-20110708-src.zip";
+                md5 = "44353697551c409c5f3207b7a87bdc22";
+              };
+  ftecommon = fetchurl {
+                url = "mirror://sourceforge/fte/fte-20110708-common.zip";
+                md5 = "0f0b6a511d5fb161235dee35df169e47";
+              };
+  src = [ ftesrc ftecommon ];
+
+  buildFlags = "PREFIX=$(out)";
+
+  installFlags = "PREFIX=$(out) INSTALL_NONROOT=1";
+
+  meta = with stdenv.lib; {
+    description = "A free text editor for developers";
+    homepage = http://fte.sourceforge.net/;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.eelco ];
```

Comment:
I don't think @edolstra has agreed to be a maintainer. Please, add yourself as a maintainer if you want.